cms/templates/js/move-xblock-list.underscore

```diff
@@ -27,7 +33,7 @@
                         </span>
                     <% } %>
                     <span class="icon fa fa-arrow-right forward-sr-icon" aria-hidden="true"></span>
-                    <span class="sr forward-sr-text"><%- forwardButtonSRText %></span>
+                    <span class="sr forward-sr-text"><%- gettext("Click for children") %></span>
```

Comment:
@catong Maybe we can use items/blocks but they seem to be generic ? @cptvitamin Any thoughts ?